lib/mongo/database/view.rb

```diff
@@ -81,9 +81,9 @@ def collection_names(options = {})
       # @return [ Array<Hash> ] Info for each collection in the database.
       #
       # @since 2.0.5
-      def list_collections
+      def list_collections(filter={})
```

Comment:
Looking at https://docs.mongodb.com/manual/reference/command/listCollections/, the command accepts 3 parameters: filter, nameOnly and authorizedCollections. To allow the driver to be future-proof, can this method take `**options` as the argument which will be passed to `collections_info`, and make `filter` one of the options? We do not need to support the other options right now.

Reply:
That makes a lot of sense. Thank you! 🙏